core-services/ci-secret-generator/_config.yaml

```diff
@@ -70,6 +70,10 @@
   attachments:
   - cmd: gcloud iam service-accounts keys create /dev/stdout --iam-account do-not-delete-ci-provisioner@openshift-gce-devel-ci.iam.gserviceaccount.com
     name: credentials.json
+- item_name: jenkins-ci-provisioner
```

Comment:
This value is almost certainly wrong.  We don't want to clobber the usual CI provisioner secret.